services/galley/test/integration/API/Teams.hs

```diff
@@ -1008,13 +1007,172 @@ testUpdateTeam = do
     checkTeamUpdateEvent tid u wsOwner
     checkTeamUpdateEvent tid u wsMember
     WS.assertNoEvent timeout [wsOwner, wsMember]
-  where
-    checkTeamUpdateEvent tid upd w = WS.assertMatch_ timeout w $ \notif -> do
-      ntfTransient notif @?= False
-      let e = List1.head (WS.unpackPayload notif)
-      e ^. eventType @?= TeamUpdate
-      e ^. eventTeam @?= tid
-      e ^. eventData @?= Just (EdTeamUpdate upd)
+
+testTeamAddRemoveMemberAboveThresholdNoEvents :: HasCallStack => TestM ()
+testTeamAddRemoveMemberAboveThresholdNoEvents = do
+  o <- view tsGConf
+  c <- view tsCannon
+  let truncationLimit = fromIntegral . fromRange $ Galley.currentTruncationLimit o
+  (owner, tid) <- Util.createBindingTeam
+  member1 <- addTeamMemberAndExpectEvent True tid owner
+
+  -- Now last fill the team until truncationSize - 2
+  let perms = Util.symmPermissions [CreateConversation]
+  replicateM_ (truncationLimit - 4) $ do
+    rand <- newTeamMember' perms <$> Util.randomUser
+    Util.addTeamMemberInternal tid rand
```

Comment:
@akshaymankar the reason why this passes the CI tests is a side effect of how `addTeamMemberInternal` works. Since our CI has maxTeamSize == truncationLimit, and when adding users internally we overcome the limit (due to mismatch on ES), then the events are anyway dropped so the test is still valid. We should change this to create "real users" in "real teams"

Reply:
https://github.com/wireapp/wire-server/commit/e7b74337dd1e925dcf54580559930c1be22ad501 - improved the test and make it run a little faster